recipes/napari-locan/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "napari-locan" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-locan-{{ version }}.tar.gz
+  sha256: 476619faccd96d847a17ddb82634b908f754bcd4d0c01af3b379f2b9d902551f
+
+build:
+  skip: true  # [py<39]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - setuptools >=61
+    - wheel
+    - setuptools-scm >=6.2
+    - pip
+  run:
+    - python
+    - locan >=0.17
+    - magicgui
+    - napari
+    - napari-matplotlib
+    - numpy
+    - qtpy
+    - tomli >=1.1.0  # [py<311]
```

Comment:
```suggestion
    - tomli >=1.1.0
```

Reply:
It is better to create a noarch package, if the only reason to not make it like that was adding this optional dependency, I suggest it is not worth it and we can leave it there as a dependency for all cases.

This will simplify packaging and produce less artifacts